src/libraries/System.Private.CoreLib/src/Internal/Console.Android.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Internal
+{
+    public static partial class Console
+    {
+        public static unsafe void Write(string s)
+        {
+            Interop.Logcat.AndroidLogPrint(Interop.Logcat.LogLevel.Debug, "DOTNET", s);
+        }
```

Comment:
The existing Console.Windows.cs and Console.Unix.cs also have `Error` nested type. It would be nice to match it.

Reply:
Done, good catch :)